crypto/crypto/src/unit_tests/ed25519_test.rs

```diff
@@ -161,7 +161,7 @@ proptest! {
         keypair in uniform_keypair_strategy::<Ed25519PrivateKey, Ed25519PublicKey>()
     ) {
         let hashable = CryptoHashable(x);
-        let signature = keypair.private_key.sign(&hashable).expect("all `RawTransaction` objects should LCS-serialize correctly");
```

Comment:
So, this was the only occurrence of the faillible `sign`? Everybody is hashing first then using sign_message, I suppose?

Reply:
All the `sign` instances added herein are still faillible.

>  Everybody is hashing first then using sign_message, I suppose?

Yes.